english/snooze.md

```diff
@@ -0,0 +1 @@
+You know the alarm in your phone is put on snooze, you go back to sleep. When alarm wake you up next time after 10 minutes everything is back to normal, you don't even remember your marshmallows dream.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.